sql/src/main/java/org/apache/druid/sql/calcite/planner/QueryHandler.java

```diff
@@ -173,10 +173,11 @@ private static RelDataType getExplainStructType(RelDataTypeFactory typeFactory)
   {
     return typeFactory.createStructType(
         ImmutableList.of(
+            Calcites.createSqlType(typeFactory, SqlTypeName.VARCHAR),
             Calcites.createSqlType(typeFactory, SqlTypeName.VARCHAR),
             Calcites.createSqlType(typeFactory, SqlTypeName.VARCHAR)
         ),
-        ImmutableList.of("PLAN", "RESOURCES")
+        ImmutableList.of("PLAN", "RESOURCES", "STATEMENT_ATTRIBUTES")
```

Comment:
I wonder if we should just call this `ATTRIBUTES`. We might want to add more info into this column in the future that might not exactly be statement-specific.

Reply:
@kfaraz, what kind of other attributes do you've in mind besides the statement-related attributes Gian mentioned in [his comment](https://github.com/apache/druid/pull/14074#pullrequestreview-1383885253)? IMO `ATTRIBUTES` seems a little too generic, but also naming is hard. :-)

Reply:
I can't currently think of what other such attributes would be, but I was thinking along the lines of Druid Catalog. When and if we do have something like catalog, the ATTRIBUTES could potentially contain some info about the table(s) used in the EXPLAIN. There could be other things too.

Also, but less importantly, it kind of fits the existing scheme of single-word nomenclature. PLAN, RESOURCES and ATTRIBUTES.

I agree that it seems too generic, but I feel that is okay, as you already explicitly specify that it is the  statementType (and say not just type ). So, ATTRIBUTES could be a free-form map where we can dump any extra info that doesn't fall into the category of PLAN or RESOURCES.

Reply:
@kfaraz, good thinking! I got convinced with the naming suggestion.